testdata/project-v3-with-deploy-image/PROJECT

```diff
@@ -0,0 +1,28 @@
+domain: testproject.org
+layout:
+- go.kubebuilder.io/v3
+plugins:
+  deploy-image.go.kubebuilder.io/v1-alpha:
+    containerCommand: memcached -m=64 modern -v
+    containerPort: "11211"
+    image: memcached:1.6.15-alpine
```

Comment:
I found a nit issue here.
The data : 

```
containerCommand: memcached -m=64 modern -v
    containerPort: "11211"
    image: memcached:1.6.15-alpine
```

Probably need to be under the resources.
What will be its results if we create 2 apis with diff images and etc?
We would need to have both resources with the options used for each scenario.

Could you please do a test and let us know?
I am ok with we solving it in a follow-up.